src/main/scala/cromwell/Main.scala

```diff
@@ -20,16 +21,39 @@ object Actions extends Enumeration {
 }
 
 object Main extends App {
-  val Properties = System.getProperties
-  val LoggerProperty = "CROMWELL_LOGGER"
-  lazy val Log = LoggerFactory.getLogger("main")
-
-  Option(Properties.getProperty(LoggerProperty)) match {
-    case None if args.headOption.map {_.toUpperCase}.contains("SERVER") =>
-      Properties.setProperty(LoggerProperty, "SERVER")
-    case _ =>
+  def initLogging() = {
```

Comment:
might as well make this private if it's only going to be used here. also a minor style thing but I find these sorts of functions are better to put after the main code which is calling it (The whole "I'll go see what it is if I care" argument)